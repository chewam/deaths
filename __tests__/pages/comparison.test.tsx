import messages from "@/lang/fr.json"
import { IntlProvider } from "react-intl"
import { render } from "@testing-library/react"

import Page from "../../src/pages/comparison"

test("Page snapshot: comparison", () => {
  jest.mock("next/router", () => ({
    useRouter() {
      return {
        locale: "fr",
        defaultLocale: "fr",
      }
    },
  }))

  const { asFragment } = render(
    <IntlProvider locale="fr" messages={messages}>
      <Page />
    </IntlProvider>
  )
  expect(asFragment()).toMatchSnapshot()
})
