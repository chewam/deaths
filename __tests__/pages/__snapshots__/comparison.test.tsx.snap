// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page snapshot: comparison 1`] = `
<DocumentFragment>
  <div
    class="container mx-auto px-6"
  >
    <div
      class="comparison"
    >
      <div
        class="chart"
      >
        <canvas
          height="0"
          role="img"
          style="display: block; box-sizing: border-box; height: 0px; width: 0px;"
          width="0"
        />
      </div>
      <ul
        class="years"
      >
        <li>
          <button
            class="active"
            style="background-color: rgb(209, 207, 226); border-color: #d1cfe2;"
          >
            2022
          </button>
        </li>
        <li>
          <button
            class="active"
            style="background-color: rgb(212, 175, 185); border-color: #d4afb9;"
          >
            2021
          </button>
        </li>
        <li>
          <button
            class="active"
            style="background-color: rgb(66, 129, 164); border-color: #4281a4;"
          >
            2020
          </button>
        </li>
        <li>
          <button
            class=""
            style="background-color: transparent;"
          >
            2019
          </button>
        </li>
        <li>
          <button
            class=""
            style="background-color: transparent;"
          >
            2018
          </button>
        </li>
        <li>
          <button
            class="active"
            style="background-color: rgb(254, 147, 140); border-color: #fe938c;"
          >
            2017
          </button>
        </li>
        <li>
          <button
            class=""
            style="background-color: transparent;"
          >
            2016
          </button>
        </li>
        <li>
          <button
            class=""
            style="background-color: transparent;"
          >
            2015
          </button>
        </li>
        <li>
          <button
            class=""
            style="background-color: transparent;"
          >
            2014
          </button>
        </li>
        <li>
          <button
            class=""
            style="background-color: transparent;"
          >
            2013
          </button>
        </li>
        <li>
          <button
            class=""
            style="background-color: transparent;"
          >
            2012
          </button>
        </li>
        <li>
          <button
            class=""
            style="background-color: transparent;"
          >
            2011
          </button>
        </li>
        <li>
          <button
            class=""
            style="background-color: transparent;"
          >
            2010
          </button>
        </li>
        <li>
          <button
            class=""
            style="background-color: transparent;"
          >
            2009
          </button>
        </li>
        <li>
          <button
            class=""
            style="background-color: transparent;"
          >
            2008
          </button>
        </li>
        <li>
          <button
            class=""
            style="background-color: transparent;"
          >
            2007
          </button>
        </li>
        <li>
          <button
            class=""
            style="background-color: transparent;"
          >
            2006
          </button>
        </li>
        <li>
          <button
            class=""
            style="background-color: transparent;"
          >
            2005
          </button>
        </li>
        <li>
          <button
            class=""
            style="background-color: transparent;"
          >
            2004
          </button>
        </li>
        <li>
          <button
            class=""
            style="background-color: transparent;"
          >
            2003
          </button>
        </li>
        <li>
          <button
            class=""
            style="background-color: transparent;"
          >
            2002
          </button>
        </li>
        <li>
          <button
            class=""
            style="background-color: transparent;"
          >
            2001
          </button>
        </li>
        <li>
          <button
            class=""
            style="background-color: transparent;"
          >
            2000
          </button>
        </li>
      </ul>
    </div>
  </div>
</DocumentFragment>
`;
